control/control.rnc

```diff
@@ -123,6 +123,7 @@ globals_elements =
     | displaymanager_shutdown
     | enable_autologin
     | enable_systemd_boot
+    | prefered_bootloader
```

Comment:
there is typo in name. It should be preferred - https://www.vocabulary.com/dictionary/preferred

Reply:
Ups, thanks. I have also fixed the code in yast-bootmanager
https://github.com/yast/yast-bootloader/pull/715